README.md

```diff
@@ -39,7 +39,79 @@ In your application.js:
 ```
 ## Usage
 
-We're not sure yet.
+Given a simple application with Tasks belonging to Projects, the first thing to do is allow your Project model to accepts_nested_attributes_for.
+
+**Models**
+
+```ruby
+class Project < ApplicationRecord
+  has_many :tasks, inverse_of: :project
+  accepts_nested_attributes_for :tasks, reject_if: :all_blank, allow_destroy: true
+end
+
+class Task < ApplicationRecord
+  belongs_to :project
+end
+```
+
+Because Rails 5 and above belongs_to associations are required by default, we must specify the link because the child object(tasks) cannot be saved until the parent (a project) is saved.
+
+Moving on, we'll need to add the parameters to the controller.
+
+**Controller**
+
+```ruby
+# ProjectsController
+
+private
+
+def project_params
+  params.require(:project).permit(:name, :description, tasks_attributes: [:id, :description, :done, :_destroy])
+end
+```
+
+Notice how we added the _destroy parameter, this is important to allow us to remove the records. Additionally, the id parameter is required.
+
+**Form**
+
+Inside of our Project form, we'll have a simple fields_for addition that will loop through all of the tasks and render a `task_fields` partial which will contain the elements used for entering a new task, or modifying an existing task.
+
+
+```erb
+<div class="form-group">
+  <%= f.label(:tasks) %>
+
+  <%= f.fields_for :tasks do |task| %>
+    <%= render 'task_fields', f: task %>
+  <% end %>
+
+  <%= f.add_association 'Add Task',
+    'tasks',
+    class: 'btn btn-secondary btn-sm' %>
+</div>
+```
+
+Additionally, we call `f.add_association` a FormBuilder method to add another field so you can add an additional task. While other gems like Cocoon take a global approach with a helper, this scopes things to forms so they are not globally available.
```

Comment:
I would add a more detailed explanation of what `add_association` is.

Also, this sentence does make sense in my head:
> Additionally, we call `f.add_association` a FormBuilder method to add another field so you can add an additional task.